CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
-Add: new functions to set callbacks for removal of devices and groups (#735)
```

Comment:
That line should be removed. Please, recover it.

Reply:
Added the above mentioned line.